src/presenter/PresenterRepository.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Vinícius Ferrão <vinicius@ferrao.net.br>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <algorithm>
+#include <cloysterhpc/presenter/PresenterRepository.h>
+
+PresenterRepository::PresenterRepository(
+    std::unique_ptr<Cluster>& model, std::unique_ptr<Newt>& view)
+    : Presenter(model, view)
+{
+    auto manager = m_model->getRepoManager();
```

Comment:
`const auto`?

Reply:
Here?

But we need mutability for the enable-disable code below:

```c++
         for (const auto& [id, _name, _state] : repo_strings) {
             manager.disable(id);
         }

         for (const auto& repo : selectedRepositories) {
             manager.enable(repo);
         }
         ```

Reply:
My bad.